assets/model_monitoring/components/pytest.ini

```diff
@@ -2,4 +2,5 @@
 [pytest]
 markers =
     e2e: mark a test as a full e2e test running on a live AML stack.
-    unit: mark a test as a unit test running in memory.
\ No newline at end of file
+    unit: mark a test as a unit test running in memory.
+    gsq_test: mark a test as a generation safety quality component test.
```

Comment:
We don't have to do it this PR but we could consider using this new gsq_test marker for our genAI signals and component e2e tests too. (token statistics, genAI preprocessor, etc)